cmd/buckets.go

```diff
@@ -11,5 +17,96 @@ func NewBucketsCommand() *cobra.Command {
 	}
 
 	ret.AddCommand(NewBucketUpgrade())
+	ret.AddCommand(NewBucketDeleteCommand())
+	ret.AddCommand(NewBucketListCommand())
+	ret.AddCommand(NewBucketRestoreCommand())
 	return ret
 }
+
+func addCommonFlags(cmd *cobra.Command) {
+	bunconnect.AddFlags(cmd.Flags())
+	otlp.AddFlags(cmd.Flags())
+	otlptraces.AddFlags(cmd.Flags())
+}
+
+func NewBucketDeleteCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "delete",
+		Short: "Delete buckets that were marked for deletion N days ago",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			days, _ := cmd.Flags().GetInt("days")
+			return withStorageDriver(cmd, func(driver *driver.Driver) error {
+				deletedBuckets, err := driver.GetBucketsMarkedForDeletion(cmd.Context(), days)
+				if err != nil {
+					return err
+				}
+				
+				for _, bucket := range deletedBuckets {
+					if err := driver.PhysicallyDeleteBucket(cmd.Context(), bucket); err != nil {
+						return err
+					}
+					cmd.Printf("Bucket %s physically deleted\n", bucket)
+				}
+				
+				return nil
+			})
+		},
+	}
+	
+	cmd.Flags().Int("days", 30, "Delete buckets marked for deletion N days ago")
```

Comment:
You could maybe use a duration instead of a number of days.
It would be more flexible.